Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -322,6 +322,17 @@ namespace AzToolsFramework
                 }
             }
 
+            if (m_parentRow && m_parentRow->m_isSceneSetting)
+            {
+                m_isSceneSetting = true;
+                setStyleSheet("QFrame {background-color: #555555; color: white;}");
+                setContentsMargins(0, 0, 8, 0);
+                if (m_treeDepth > 0)
+                {
+                    m_treeDepth = m_treeDepth - 1;
+                }
+            }
+
```

Comment:
is there any way to make these changes from the asset inspector using a style sheet instead of adding this code to the property row widget? I don't think it should be aware of scene settings.

Reply:
I'm not sure where else I could make these changes outside of the property row widget. The scene settings window doesn't use component cards, and the RPE and property row widget treat the reflection data as it would any normal group. To make each group look like a component card, the entire row needs to be painted a different background color and have one indentation level removed.

I'm open to discussing this further and finding a solution for a different place to make these changes, but I'm not sure where else in the scene settings this could be done.